index.html

```diff
@@ -63,6 +65,19 @@ <h1>tsParticles 404 Templates</h1>
         </div>
       </div>
     </div>
+     <div class="container">
+      <footer class="py-3 my-4">
+        <p class="text-center text-muted">Connect Us On:</p>
+        <ul class="nav justify-content-center border-bottom pb-3 mb-3 mynav">
+          <li class="nav-item"><a href="https://github.com/matteobruni/tsparticles" class="nav-link px-2 text-muted">Github</a></li>
```

Comment:
I think adding target="_blank" to links is better, so the user remains on the page